src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4748,6 +4830,23 @@ components:
             validators:
               $ref: "#/components/schemas/ValidatorUptimeCollection"
 
+    ResourceOwnersRequest:
+      allOf:
+      - $ref: "#/components/schemas/CursorLimitMixin"
+      - type: object
+        properties:
+          resource_address:
```

Comment:
@dhedey is it ok to query by just one resource? or should that endpoint support providing multiple resource addresses?

Reply:
Yeah, only one resource is fine. I think it only really makes sense to ask the question for each resource separately.